_includes/home-slides.html

```diff
@@ -1,16 +1,15 @@
-<div class="content home-slides">
-
+<div class="content home-slides p-1 pt-5 pb-2">
     <div class="row justify-content-around">
-        <div class="col-lg-8 p-2 owl-carousel">
+        <div class="col-lg-8 owl-carousel">
 
             <!-- slide -->
             <div class="row justify-content-around">
-                <div class="col-8 col-md-4  order-md-last">
+                <div class="col-8 col-md-3 order-md-last pb-4">
                     <img src="assets/img/slide_mission.png" alt="Tiendatek">
                 </div>
-                <div class="col-6">
+                <div class="col-8">
```

Comment:
The sum of all the cols inside a row has to be 12 (except in galleries and stuff like that). Quite weird having 2 cols of 8, one of them defining `col-md` and the other not... Do you need help understanding this?

Reply:
@durey Ready!